app/views/collection_drafts/forms/fields/_doi_fields.html.erb

```diff
@@ -1,20 +1,59 @@
-<!-- DOI -->
-<%= mmt_text_area(
-  name: 'doi',
-  title: 'DOI',
-  prefix: name_prefix,
-  value: doi['DOI'],
-  help: 'definitions/DoiType/properties/DOI',
-  validate: true,
-  required: true
- ) %>
+<!-- doi available? -->
+<div class="doi-group">
+  <% doi_available = doi.nil? ? true : !doi['DOI'].nil? %>
```

Comment:
so we're checking if both draft.draft['DOI'] _and_ draft.draft['DOI']['DOI'] are nil?

Reply:
right, `draft['DOI']` should be an object, so if it is missing altogether I wanted to pre-select the Available radio button. If `draft['DOI']` is an object, the Available radio button needs to be selected if `draft['DOI']['DOI']` is populated